glossary.yml

```
@@ -393,6 +420,11 @@
     def: >
       A way to control the appearance of HTML.  CSS is typically used to specify
       fonts, colors, and layout.
+   fr:
+    term: "Feuilles de Style en Cascade"
+    def: >
+      Une manière de contrôler l'apparence du rendu HTML. Le CSS est généralement utilisé dans le but
+      de spécifier la police, la couleur ainsi que la structure d'une page web.
```

peut-être utiliser le pluriel car c'est rare qu'une seule police/couleur soit utilisée ?

```suggestion
      de spécifier les polices, les couleurs ainsi que la structure d'une page web.
```